wgpu-core/src/command/compute.rs

```diff
@@ -791,6 +810,11 @@ fn set_push_constant(
         end_offset_bytes,
     )?;
 
+    let offset_in_elements = (offset / wgt::PUSH_CONSTANT_ALIGNMENT) as usize;
+    let size_in_elements = (size_bytes / wgt::PUSH_CONSTANT_ALIGNMENT) as usize;
+    state.push_constants[offset_in_elements..offset_in_elements + size_in_elements]
+        .copy_from_slice(data_slice);
```

Comment:
nitpick: There's a (slightly) more brain-friendly way to write this!

```suggestion
    state.push_constants[offset_in_elements..][..size_in_elements].copy_from_slice(data_slice);

```

Reply:
Implemented as suggested with a force-push to c3d50b89.